cmd/temporalite/main.go

```diff
@@ -162,6 +163,10 @@ func buildCLI() *cli.App {
 					return cli.Exit(fmt.Sprintf("ERROR: only one of %q or %q flags may be passed at a time", ephemeralFlag, dbPathFlag), 1)
 				}
 
+				if err := os.MkdirAll(filepath.Dir(c.String(dbPathFlag)), os.ModePerm); err != nil {
```

Comment:
I think we only want to do this when the user didn't provide their own path

Reply:
Done